AbletonMCP_Remote_Script/handlers/session.py

```
@@ -0,0 +1,1130 @@
+
+def set_song_loop(song, enabled, start, length, ctrl=None):
+    """Control arrangement loop bracket."""
+    try:
+        if enabled is not None:
+            song.loop = bool(enabled)
+        if start is not None:
+            song.loop_start = max(0.0, float(start))
+        if length is not None:
+            length_val = float(length)
+            if length_val <= 0:
+                raise ValueError("Loop length must be positive, got {0}".format(length_val))
```

Ruff (0.15.0)

[warning] Abstract `raise` to an inner function

(TRY301)